spec/features/collection_search_results_spec.rb

```diff
@@ -13,7 +13,7 @@
     it 'is active when collection search is activated' do
       visit search_catalog_path f: { level_sim: ['Collection'] }, search_field: 'all_fields'
       within '#main-container' do
-        expect(page).to have_css '.al-collection-count', text: /\d+ collections/
```

Comment:
I'm wondering if we can just delete this entire file now?

Reply:
Yeah, that probably a better idea. Will do!

Reply:
1. Done!